tempesta_fw/http_parser.c

```diff
@@ -1351,6 +1353,7 @@ __req_parse_cache_control(TfwHttpReq *req, unsigned char *data, size_t len)
 	__FSM_STATE(Req_I_CC_m) {
 		TRY_STR("max-age=", Req_I_CC_MaxAgeV);
 		TRY_STR("min-fresh=", Req_I_CC_MinFreshV);
+		TRY_STR("max_stale", Req_I_CC_MaxStale);
```

Comment:
RFC 7234 5.2.1.2 doesn't say anything about "max**_**stale" (`_` instead of `-`).